rustls-libssl/src/entry.rs

```diff
@@ -131,81 +128,67 @@ entry! {
 entry! {
     pub fn _SSL_CTX_get_options(ctx: *const SSL_CTX) -> u64 {
         let ctx = try_clone_arc!(ctx);
-        ctx.lock()
-            .ok()
-            .map(|ctx| ctx.get_options())
-            .unwrap_or_default()
+        ctx.get().get_options()
```

Comment:
nit: I think at this point the `ctx` binding ends up being noise. WDYT about collapsing instances like this to:

```rust
entry! {
    pub fn _SSL_CTX_get_options(ctx: *const SSL_CTX) -> u64 {
        try_clone_arc!(ctx).get().get_options()
    }
}
```

I won't flag each individually assuming that if you agree you will do a sweep.

Reply:
Have done a sweep of these and addressed the vast majority. The ones I've left in are where I'd like to keep the basic null parameter checking above any real work or other errors -- there are only a few of these.

Reply:
LGTM!